src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -50,20 +50,24 @@ export class BiggiveMainMenu {
   }
 
   private setHeaderSize() {
-    if (this.host.offsetHeight === this.lastOffsetHeight) {
+    if (this.host.scrollHeight === this.lastScrollHeight) {
       // Some browsers fire 'resize' overzealously on scroll; we don't want to cause extra paints if nothing
       // relevant changed.
       return;
     }
 
-    this.lastOffsetHeight = this.host.offsetHeight;
+    this.lastScrollHeight = this.host.scrollHeight;
 
     // Some resize edge cases lead Firefox, and maybe others, to go haywire and get a host offset
     // height of millions of pixels, presumably due to a layout logic loop. So for as long as we use
     // this body padding workaround, we need a safe maximum value, currently 130px, beyond which
     // we will never further displace the main content.
-    const offsetHeight = isNaN(this.host.offsetHeight) ? 0 : this.host.offsetHeight;
-    document.body.style.paddingTop = Math.min(130, offsetHeight).toString() + 'px';
+    // (Possibly scrollHeight could have the same issue, not tested.)
+    const scrollHeight = isNaN(this.host.scrollHeight) ? 60 : this.host.scrollHeight;
```

Comment:
This is the cheat to stop us ever setting a zero offset. Hopefully that has the effect of making an invisible ticker on mobile impossible. I checked desktop latest Firefox and latest Safari and both had exactly 130px menu height at desktop sizes and 60px at mobile sizes, so hopefully a 60px minimum is a fairly small risk for adding superfluous space in modern browsers.